src/somesy/core/writer.py

```diff
@@ -249,6 +264,9 @@ def sync(self, metadata: ProjectMetadata) -> None:
         self.maintainers = self._sync_person_list(
             self.maintainers, metadata.maintainers()
         )
+        # self.contributors = self._sync_person_list(
```

Comment:
should delete commented out code?

Reply:
that's actually a point maybe needs a bit of discussion. 

In `pom.xml` there's developers and contributors, and contributors is apparently for minor stuff (compared to developers), from the POM reference:

```
Contributors are like developers yet play an ancillary role in a project's lifecycle. Perhaps the contributor sent in a bug fix, or added some important documentation. A healthy open source project will likely have more contributors than developers.
```

In `package.json` you apparently dump **all** people, and just a single author is highlighted. So I'm not fully sure whether and how these semantics can be made to match and how best represent it in code.

If we could harmonize it, maybe having the contributors handled in the core/writer class would make sense... but I guess for now I'll remove the commented part.